dts/bindings/display/raydium,rm68200.yaml

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2022, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: Raydium RM68200 Panel
+
+compatible: "raydium,rm68200"
+
+include: mipi-dsi-device.yaml
+
+properties:
+    label:
+      required: true
+
+    reset-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        The RESETn pin is asserted to disable the sensor causing a hard
+        reset.  The sensor receives this as an active-low signal.
+
+    pwr-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        The PWRn pin is asserted to do a power on for the panel.
+        The panel receives this as an active-high signal.
+
+    bl-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        The BLn pin is asserted to control the backlight of the panel.
+        The sensor receives this as an active-high signal.
+
+    height:
+      type: int
+      required: true
+      description:
+        Height of the panel driven by the controller, with the units in pixels.
+
+    width:
+      type: int
+      required: true
+      description:
+        Width of the panel driven by the controller, with the units in pixels.
```

Comment:
please make use of `display-controller.yaml`, these are already defined.

Reply:
Thank you.